server/src/Service/InteractionService.php

```diff
@@ -77,8 +80,12 @@ public function getInteraction(
 	 */
 	private function getDefaultStartingDate( array $users ) {
 		$dates = $this->revisionDao->getUsersFirstEditDate( $users );
-
-		return ( !$dates ) ? null : strtotime( max( $dates ) . '-1 day midnight' );
+		$ts = ( !$dates ) ? null : strtotime( max( $dates ) . '-1 day midnight' );
+		if ( $ts ) {
+			$date = new \DateTime();
```

Comment:
Use https://secure.php.net/manual/en/datetime.createfromformat.php with `U` as the format. :)

But honestly, I would create the datetime object first, then use the `sub()` method to substract the interveral:
https://secure.php.net/manual/en/datetime.sub.php
or you could use diff to get the previous day at midnight:
https://secure.php.net/manual/en/datetime.diff.php
example:
https://stackoverflow.com/a/22535685/864374

Only reason I bring this up is because `strtotime` is not very reliable, so imho, better to avoid it and just use `DateTime` wherever possible.